### DISCUSSION TITLE
Slowness in ubuntu server

### DISCUSSION THREAD

**Discussion by iwbrhwfy** [Chia-Network/chia-blockchain#6148] [Q&A]
I found that my chia plot create command running on my ubuntu server 2004 machine is much slower than windows client. Of course , two machine have exactly the same hardware in ervery aspect. Especially from phase 2. I also found that specified thread allocated to generate chia plots often gets stuck in progress. When I use htop command to list them, i always found some thread gets status "D", and then cpu usage went down dramatically. Please help me to solve my problem. I wonder that whether this is a defect on ubuntu server system only or my own configuation mistake. Thank you.
My command -- "chia plots create -k 32 -e -r 12 -n 1 -t /chia/tmp1 -d /chia/dst1"
<img width="1536" alt="截屏2021-05-28 上午11 37 18" src="https://user-images.githubusercontent.com/61858690/119927356-de034080-bfab-11eb-804d-1913548a68dc.png">

**Comment by tiny0little** [comment]
let's see - you run 11 parallel plots, each uses 12 threads, => total 132 threads. CPU might struggle with that? can TEMP location handle that? 11 * 3.6GB = 40GB RAM needed to run that, you have 32GB


>  especially from phase 2

only phase-1 uses multithreading at the moment


> ```chia plots create -k 32 -e```

why bitfield disabled? usually it improves things on old hardware

**Reply by iwbrhwfy** [reply]
> let's see - you run 11 parallel plots, each uses 12 threads, => total 132 threads. CPU might struggle with that? can TEMP location handle that? 11 * 3.6GB = 40GB RAM needed to run that, you have 32GB
> 
> > especially from phase 2
> 
> only phase-1 uses multithreading at the moment
> 
> > `chia plots create -k 32 -e`
> 
> why bitfield disabled? usually it improves things on old hardware

sorry, but I just run 1 parallel plot as the command I list before. What you saw is htop command shows that ervery thread working details but not 12 prarallel plots. My cpu has 8 cores and 16 threads and 32Gib ram, so I dont know where is the mistake, thank you for reply and any help would be thankful~!

**Reply by tiny0little** [reply]
i still think these are 11 `chia plots create` processes, as they have different PIDs, each uses 12 threads. good luck with your experiments!

**Reply by iwbrhwfy** [reply]
> i still think these are 11 `chia plots create` processes, as they have different PIDs, each uses 12 threads. good luck with your experiments!

https://superuser.com/questions/118086/why-are-there-many-processes-listed-under-the-same-title-in-htop
